types/openrct2/package.json

```diff
@@ -1,23 +1,25 @@
 {
-    "private": true,
-    "name": "@types/openrct2",
-    "version": "0.3.9999",
-    "nonNpm": true,
-    "nonNpmDescription": "OpenRCT2",
-    "projects": [
-        "https://openrct2.io/"
-    ],
-    "devDependencies": {
-        "@types/openrct2": "workspace:."
+  "private": true,
+  "name": "@types/openrct2",
+  "version": "0.4.9999",
+  "nonNpm": true,
+  "nonNpmDescription": "OpenRCT2",
+  "projects": ["https://openrct2.io/"],
+  "devDependencies": {
+    "@types/openrct2": "workspace:."
+  },
+  "owners": [
+    {
+      "name": "OpenRCT2",
+      "githubUsername": "OpenRCT2"
     },
-    "owners": [
-        {
-            "name": "OpenRCT2",
-            "githubUsername": "OpenRCT2"
-        },
-        {
-            "name": "Ted John",
-            "githubUsername": "IntelOrca"
-        }
-    ]
+    {
+      "name": "Ted John",
+      "githubUsername": "IntelOrca"
+    },
+    {
+      "name": "Tom Scott",
```

Comment:
sorry about the weird diff here. @IntelOrca i added myself as an owner so i could help out releasing new versions and contributing the `.d.ts` myself...if this is not cool let me know and i can remove.